src/messaging/ReliableMessageManager.h

```diff
@@ -57,9 +57,9 @@ class ReliableMessageManager
     {
         RetransTableEntry();
 
-        ReliableMessageContext * rc;   /**< The context for the stored CHIP message. */
-        System::PacketBuffer * msgBuf; /**< A pointer to the PacketBuffer object holding the CHIP message. */
-        uint32_t msgId;                /**< The message identifier of the CHIP message awaiting acknowledgment. */
+        ReliableMessageContext * rc;       /**< The context for the stored CHIP message. */
+        System::PacketBufferHandle msgBuf; /**< A pointer to the PacketBuffer object holding the CHIP message. */
```

Comment:
```suggestion
        System::PacketBufferHandle msgBuf; /**< A handle to the PacketBuffer object holding the CHIP message. */
```

Reply:
Done. I think I've left more of these elsewhere, and have put on my to-do list to check for them in my next cleanup PR.